tools/vsextension/ProjectTemplates/oeenclave-linux/MyEnclave.vcxproj

```diff
@@ -0,0 +1,184 @@
+  </PropertyGroup>
+  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.props" />
+  <ImportGroup Label="ExtensionSettings">
+  </ImportGroup>
+  <PropertyGroup Label="UserMacros" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x86'">
+    <GenerateManifest>false</GenerateManifest>
+    <LinkIncremental>false</LinkIncremental>
+    <IncludePath>$(NoInherit)</IncludePath>
+    <LibraryPath>$(NoInherit)</LibraryPath>
+    <RemoteCCompileToolExe>clang-7</RemoteCCompileToolExe>
+    <RemoteCppCompileToolExe>clang-7</RemoteCppCompileToolExe>
+    <RemoteLdToolExe>clang-7</RemoteLdToolExe>
+    <RemoteLdCommmandTimeout />
+    <TargetExt />
+    <TargetName>$(ProjectName)</TargetName>
+    <RemoteTargetPath>$(RemoteOutDir)/$(TargetName)</RemoteTargetPath>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x86'">
+    <GenerateManifest>false</GenerateManifest>
+    <LinkIncremental>false</LinkIncremental>
+    <IncludePath>$(NoInherit)</IncludePath>
+    <LibraryPath>$(NoInherit)</LibraryPath>
+    <RemoteCCompileToolExe>clang-7</RemoteCCompileToolExe>
+    <RemoteCppCompileToolExe>clang-7</RemoteCppCompileToolExe>
+    <RemoteLdToolExe>clang-7</RemoteLdToolExe>
+    <RemoteLdCommmandTimeout />
+    <TargetExt />
+    <TargetName>$(ProjectName)</TargetName>
+    <RemoteTargetPath>$(RemoteOutDir)/$(TargetName)</RemoteTargetPath>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
+    <GenerateManifest>false</GenerateManifest>
+    <LinkIncremental>false</LinkIncremental>
+    <IncludePath>$(NoInherit)</IncludePath>
+    <LibraryPath>$(NoInherit)</LibraryPath>
+    <RemoteCCompileToolExe>clang-7</RemoteCCompileToolExe>
+    <RemoteCppCompileToolExe>clang-7</RemoteCppCompileToolExe>
+    <RemoteLdToolExe>clang-7</RemoteLdToolExe>
+    <RemoteLdCommmandTimeout />
+    <TargetExt />
+    <TargetName>$(ProjectName)</TargetName>
+    <RemoteTargetPath>$(RemoteOutDir)/$(TargetName)</RemoteTargetPath>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
+    <GenerateManifest>false</GenerateManifest>
+    <LinkIncremental>false</LinkIncremental>
+    <IncludePath>$(NoInherit)</IncludePath>
+    <LibraryPath>$(NoInherit)</LibraryPath>
+    <RemoteCCompileToolExe>clang-7</RemoteCCompileToolExe>
+    <RemoteCppCompileToolExe>clang-7</RemoteCppCompileToolExe>
+    <RemoteLdToolExe>clang-7</RemoteLdToolExe>
+    <RemoteLdCommmandTimeout />
+    <TargetExt />
+    <TargetName>$(ProjectName)</TargetName>
+    <RemoteTargetPath>$(RemoteOutDir)/$(TargetName)</RemoteTargetPath>
+  </PropertyGroup>
+  <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x86'">
+    <ClCompile>
+      <Optimization>Disabled</Optimization>
+      <WarningLevel>Level3</WarningLevel>
+      <PreprocessorDefinitions>_WINDLL;%(PreprocessorDefinitions);OE_ECALL=</PreprocessorDefinitions>
```

Comment:
Is `_WINDLL` necessary for the Linux project? `OE_ECALL` is no longer in use in the SDK.